playground/AWS/AWS.AppHost/Program.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Amazon;
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+// Setup a configuration for the AWS .NET SDK.
+var awsConfig = builder.AddAWSSDKConfig()
+                        .WithProfile("default")
+                        .WithRegion(RegionEndpoint.USEast2);
+
+// Provision application level resources like SQS queues and SNS topics defined in the CloudFormation template file app-resources.template.
+var awsResources = builder.AddAWSCloudFormationTemplate("AspireSampleDevResources", "app-resources.template")
+                        // Add the SDK configuration so the AppHost knows what account/region to provision the resources.
+                        .WithReference(awsConfig);
+
+builder.AddProject<Projects.Frontend>("frontend")
+        // Reference the CloudFormation resource to project. The output parameters will added to the IConfiguration of the project.
+        .WithReference(awsResources)
+        // Assign the SDK config to the project. The service clients created in the project relying on environment config
+        // will pick up these configuration.
+        .WithReference(awsConfig);
```

Comment:
Do apps still need this information to connect to the resources?

Reply:
That isn't encapsulated in the output values?

Reply:
Yes the awsConfig identifies what AWS profile credentials the SDK's service clients should use. The output values are really freeform but usually the identifiers of the resources created in the CloudFormation stack.

The `awsConfig` information would not be used during deployment because the compute environment in AWS would provide credentials and region information.

Reply:
This is very different to azure. Does that imply that resource names are not global but tied to a specific "profile"?

Reply:
A profile is a local configuration in your `~/.aws/credentials` file that provides either static AWS credentials or config on how to acquire credentials like an SSO or MFA. A profile does not equal an account it is your local config that provides information to acquire credentials for an account.

You can have multiple profiles in your credentials files pointing to the same account if you want. Maybe for your AppHost you configure with a "dev" profile with restricted permissions but you also might have an "admin" profile to work outside of AppHost. 

My limited understanding of Azure I believe this is an area they are very different with Azure having subscriptions and you choosing the subscription you want to use. I'm probably completely wrong about that 😄